cocos/2d/CCParticleSystem.cpp

```diff
@@ -839,6 +839,8 @@ void ParticleSystem::update(float dt)
         if (_particleCount < _totalParticles)
         {
             _emitCounter += dt;
+            if (_emitCounter < 0.f)
```

Comment:
In SchedulerTest, ControlSlider* SchedulerTimeScale::sliderCtl() initialized the slider bar from -3.0 to 3.0, when drag the slider bar to left, Director::getInstance()->getScheduler()->setTimeScale will send a param with negative value, then scheduler will call all update registered in it with negative value, this make particle emitCounter & elapsed been negative then crash render.